README.md

```diff
@@ -85,7 +84,7 @@ Here is the default setup function. All of these values are optional, and if you
 
 ```lua
 require("gitlab").setup({
-  port = 21036, -- The port of the Go server, which runs in the background
+  port = 21036, -- The port of the Go server, which runs in the background, if omitted random port will be chosen automatically.
```

Comment:
Let's write this as `nil` in the documentation since that's what's used by default:

```
 port = nil, -- The port of the Go server, which runs in the background, if omitted or `nil` the port will be chosen automatically
```

Reply:
you mean `nil` in comment or instead of value 21036?